app/assets/stylesheets/homepage.css.scss

```diff
@@ -36,7 +36,7 @@
 
 #logo {
     text-decoration: none;
-    margin-left: -40px ;
+    margin-left: -70px ;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Declaration should not have a space before the terminating semicolon